skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
- Add a description of any changes made to the original bx-python code
- Make a note that this code was taken from bx-python, and reference licenses/bx_python.txt